src/admin/dashboard/pages/reverse-withdrawal/ReverseWithdrawalTransaction.tsx

```
@@ -0,0 +1,520 @@
+    }, [] );
+
+    useEffect( () => {
+        fetchTransactions();
+    }, [ fetchTransactions ] );
+
+    const clearSingleFilter = ( filterId ) => {
+        const args = { ...filterArgs };
+        switch ( filterId ) {
+            case 'date-range':
+                // Reset to default 30 days window like Vue
+                const end = new Date();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)